### ISSUE TITLE
Results and Rankings is displayed for zero seconds when "Show intro upon field activation".

### ISSUE THREAD

**Issue by paulsp** [brenapp/tm-switcher#22]
tm_switcher v1.7.0 

Both the results and ranking are displayed for zero seconds when "Show intro upon field activation".  Although the next field is activated when the match on the current field ends, is there a way to display the results and rankings for a minimum number of seconds while the field is active?  Once the number of seconds, the display would switch to "INTRO".  Granted once the match starts, the display should change to "IN_MATCH"

```
? VEX TM Address: 127.0.0.1
? Would you like to control OBS? Yes
? OBS Websocket Address: ws://127.0.0.1:4455
? OBS Websocket (leave blank for no password): 
? Would you like to control an ATEM device over the network? No

Connecting to servers: 
✅ Tournament Manager
✅ OBS

? Attach to which fieldset?  Match Field Set #1
? What OBS scene do you want to associate with Red Field?  Red Field
? What OBS scene do you want to associate with Green Field?  Green Field
? What OBS scene do you want to associate with Blue Field?  Blue Field

? Which audience display automation would you like to enable? Show intro upon field activation, Show saved score after match, Show rankings after every 6th match
? Start and stop recording for each match?  No
```
```
[2023-12-04T02:36:03.486Z] info: Match ended, showing saved match results
[2023-12-04T02:36:03.501Z] info: [00:00:00] Q 2 ended on field 2 
[2023-12-04T02:36:03.502Z] info: [00:00:00] Audience display changed to RESULTS
[2023-12-04T02:36:03.504Z] info: Field activated, showing intro
[2023-12-04T02:36:03.508Z] info: [00:00:00] info: Q 3 assigned to field 3 [OBS: Blue Field, ATEM: none]
[2023-12-04T02:36:03.509Z] info: [00:00:00] Q 3 activated on field 3 [OBS: Blue Field, ATEM: none]
[2023-12-04T02:36:03.514Z] info: [00:00:00] Audience display changed to INTRO
[2023-12-04T02:36:22.069Z] info: [00:00:00] Audience display changed to IN_MATCH
```
```
[2023-12-04T02:37:43.147Z] info: [00:00:00] Q 6 ended on field 3 
[2023-12-04T02:37:43.159Z] info: Field activated, showing intro
[2023-12-04T02:37:43.164Z] info: [00:00:00] Audience display changed to RANKINGS
[2023-12-04T02:37:43.164Z] info: [00:00:00] info: Q 7 assigned to field 1 [OBS: Red Field, ATEM: none]
[2023-12-04T02:37:43.164Z] info: [00:00:00] Q 7 activated on field 1 [OBS: Red Field, ATEM: none]
[2023-12-04T02:37:43.164Z] info: [00:00:00] Audience display changed to INTRO
```

**Comment by brenapp** [comment]
I have noticed this as well. This is due to Tournament Manager's new automatic queuing system, which queues the next match when the previous match ends. If you do not have V5 Field Control enabled, it will automatically send the field activation event (which is what I am listening for) to show the queue. 

If you are using V5 Field Control, the field activation event shouldn't occur until you activate the field, which mitigates this problem. However, I want a better solution for local testing and events that do not use V5 Field Control.

**Comment by brenapp** [comment]
Fixed by cfd2fbba676d16566c0bc7ea28a6c52ef1a82ddd

**Comment by paulsp** [comment]
This was noticed while doing table top testing without V5 Smart Field controllers attached. My tournaments use V5 SmartField controllers, so I should include them when testing to better understand the behavior.
